libvast/src/system/application.cpp

```diff
@@ -72,7 +72,8 @@ auto make_root_command(std::string_view path) {
       .add<std::vector<std::string>>(
         "schema-paths", "list of paths to look for schema files "
                         "([" VAST_INSTALL_PREFIX "/share/vast/schema])")
-      .add<std::string>("directory,d", "directory for persistent state")
+      .add<std::string>("db-directory", "directory for persistent state")
+      .add<std::string>("log-directory", "directory for log files")
```

Comment:
Maybe we can use `-D` and `-L` (or lowercase) as shortcuts? I would imagine that this comes in handy for testing.

Reply:
Let's keep `-d` as a short option for `--db-directory` to avoid breaking scripts.

Reply:
Then `-l` for *l*og directory makes more sense.

Reply:
I've added the short options.